Makefile.armv7a

```diff
@@ -13,9 +13,17 @@ MKDEPFLAGS = $(CFLAGS)
 
 CC = $(CROSS)gcc
 
-CFLAGS += -Os -Wall -Wstrict-prototypes -g\
-	-mcpu=cortex-a7 -mtune=cortex-a7 -mfpu=neon-vfpv4 -mfloat-abi=hard -mthumb\
-	-fomit-frame-pointer -mno-unaligned-access -fdata-sections -ffunction-sections
+ifeq ($(TARGET_FAMILY), armv7a7)
+	CFLAGS += -Os -Wall -Wstrict-prototypes -g\
+		-mcpu=cortex-a7 -mtune=cortex-a7 -mfpu=neon-vfpv4 -mfloat-abi=hard -mthumb\
+		-fomit-frame-pointer -mno-unaligned-access -fdata-sections -ffunction-sections
+else ifeq ($(TARGET_FAMILY), armv7a9)
+	CFLAGS += -Os -Wall -Wstrict-prototypes -g\
+		-mcpu=cortex-a9 -mtune=cortex-a9 -mfpu=neon-vfpv4 -mfloat-abi=hard -mthumb\
+		-fomit-frame-pointer -mno-unaligned-access -fdata-sections -ffunction-sections
+endif
```

Comment:
Maybe we could change only `mcpu `and `mtune `per CPU and init rest of the `CFLAG `without `ifeq`?

Reply:
Thanks for advice, done.